qiskit_experiments/database_service/json.py

```diff
@@ -54,7 +69,13 @@ def object_hook(self, obj):
                 return np.array(obj["__value__"])
             if obj["__type__"] == FitVal.__name__:
                 return FitVal(*obj["args"])
```

Comment:
If you make my suggested change above, this will become `FitVal(**obj["args"])`.